Resources/public/js/views/services/ez-contenteditviewservice.js

```diff
@@ -400,6 +400,10 @@ YUI.add('ez-contenteditviewservice', function (Y) {
         _redirectToAttribute: function (attr) {
             var attrRedirectionUrl = this.get(attr);
 
+            this.set('publishRedirectionUrl');
```

Comment:
I'm a bit confused by those 3 lines.
First because, it seems you set the value of those attributes to `undefined` while they expect either a function or string (ok that's badly documented), second because you *set* those 3 attributes while `_redirectToAttribute` expects one of those attribute identifier and last because you do after getting the actual value of the attribute. So clearly, something is wrong here.